### ISSUE TITLE
There is a problem with connecting to your Elasticsearch host

### ISSUE THREAD

**Issue by maurokh** [10up/ElasticPress#1058]
Have you searched for similar issues before submitting this one?
yes

**Is this a bug, question or feature request?**
Question

**Describe the issue you encountered:**
There is a problem with connecting to your Elasticsearch host. I have configured properly elasticSearch on the WP-config file, and installed and ran the elasticSearch service on DigitalOcean with success, but now when I try to connect elasticPress to the server, it shows the error above.

My wordpress installation is on a different server on liquid web.

**Current WordPress version:** 4.8.6

**Current ElasticPress version:** 2.4.2

**Current Elasticsearch version:** 6.1.4

**Where do you host your Elasticsearch server:** Digital Ocean

**Other plugins installed (WooCommerce, Simple Redirect Manager, etc..):**

Advanced Ads | 1.8.26
Advanced Ads Pro | 2.0.4
Advanced Ads – Anuncios PopUps y Ventanas Emergentes | 1.6
Advanced Ads – Anuncios Responsables | 1.7
Advanced Ads – Rastreador de Anuncios | 1.8.7
Advanced Ads – Visual Composer | 1.0.3
Code Snippets | 2.10.1.1
Custom Login | 3.2.8
Debug Bar | 0.9
Easy Responsive Tabs | 3.1
Galería de Canal de Youtube | 2.4
Google Analytics Dashboard for WP (GADWP) | 5.3.2
Insert Headers and Footers | 1.4.2
Lightbox para Galerías | 0.9.1
MetaSlider | 3.7.0
reSmush.it Image Optimizer | 0.1.13
Slider Revolution | 5.4.7.1
Smart Photo Gallery | 1.4.2
Tabby Responsive Tabs | 1.2.3
Widgets On Pages | 1.4.0
Wordfence Security | 7.1.2
WordPress Popular Posts | 4.0.13
WP Disable | 1.5.15
WP Rocket | 2.11.7
WPBakery Page Builder | 5.4.5
WPML Multilingual CMS | 3.9.3
WPML String Translation | 2.7.3
Yoast SEO | 6.3.1
Youtube Playlist Video Player | 1.3.1

**Steps to reproduce:**

NA

**Screenshots, if needed:**

![image](https://user-images.githubusercontent.com/38408177/38783705-67febaa2-40cb-11e8-82fe-69f8f76bcf80.png)

![image](https://user-images.githubusercontent.com/38408177/38783733-c147835a-40cb-11e8-9240-b92787c61da7.png)

**Comment by maurokh** [comment]
Hi, I'm sorry I was very sick and couldn't follow up on this. But I'm back, I appreciate your answer.

I'm sure the IP is fine and I haven't setup any authentication, I did run a curl to make sure the IP is responding:

![image](https://user-images.githubusercontent.com/38408177/39849666-25beb424-53d3-11e8-8b9e-3509c53221c9.png)

It sent me back the info related to elastic search running properly.

I tried changing network.host to 0.0.0.0 as I found it will allow any call, just to figure out if it's responding, but it is not.

I also found some steps related to bootstrap checks that need to be changed on jvm.options and I will test, but any other guidance on what other thing I could test I will totally appreciate it. 

Thank you,
